src/containers/Inventory.js

```diff
@@ -1,10 +1,12 @@
 import React from "react";
 import { connect } from 'react-redux'
-import { changeBlockName, toggleProperties } from '../actions'
+import { changeBlockName, toggleProperties, selectInventoryBlock } from '../actions'
```

Comment:
React is smart enough to import from index.js if you don't specify a specific js in actions?  I'm assuming so, since it works -- but just wanted to verify my assumption is correct.